session.c

```diff
@@ -407,12 +406,28 @@ int do_exec_windows(Session *s, const char *command, int pty) {
 		if (command[1] == ':') /* absolute */
 			exec_command = xstrdup(command);
 		else {/*relative*/
-			exec_command = malloc(strlen(progdir) + 1 + strlen(command));
+			const int command_len = strlen(progdir) + 1 + strlen(command) + 2;
+			exec_command = malloc(command_len);
 			if (exec_command == NULL)
 				fatal("%s, out of memory", __func__);
-			memcpy(exec_command, progdir, strlen(progdir));
-			exec_command[strlen(progdir)] = '\\';
-			memcpy(exec_command + strlen(progdir) + 1, command, strlen(command) + 1);
+						
+			char *c = exec_command;
+			memcpy(c, progdir, strlen(progdir));
+			c += strlen(progdir);
+			*c++ = '\\';
+
+			if(IS_INTERNAL_SFTP(command)) {
+				const char *s = "sftp-server.exe";
```

Comment:
As we discussed in windows we need to run sftp-server as separate process.. Added more comments in code and in the issue #757